.storybook/preview.js

```diff
@@ -27,7 +27,15 @@ addParameters({
   },
   options: {
     storySort: {
-      order: ['Getting Started', 'Components', ['UI', 'App'], 'Pages'],
+      order: [
+        'Getting Started',
+        'Design Tokens',
+        ['Color'],
+        ['Getting Started', 'Light Theme Colors', 'Dark Theme Colors'],
+        'Components',
+        ['UI', 'App'],
+        'Pages',
+      ],
     },
   },
 });
```

Comment:
Reordering stories so "Design Tokens" comes after "Getting Started" and doesn't get pushed to the bottom